### PR TITLE
make suggestion for upgraded README

### PR DESCRIPTION
**Pull request by Mackiovello**
For StarcounterApps/SignIn#91

The link to SignIn's README does not work yet. I'll change the headline in the SignIn app so that it works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ Address book. Manage contact information and define relations between **people**
 
 ## Requirements
 
-Requires Starcounter version 2.3.0.6067
```
Comment (warpech) [inline README.md]: The above line is generated automatically. If we want to change the text of this line, also change the match pattern in https://github.com/StarcounterApps/People/blob/ada78784e84caf44bd7c52056403f4cede8967bf/Gruntfile.js#L27

File: README.md [hunk 2]
```diff
@@ -4,9 +4,15 @@ Address book. Manage contact information and define relations between **people**
...
+For this app to run properly, a couple of things are needed:
 
-This app requires [SignIn](https://github.com/starcounterapps/signin) 4.0.0 or newer and the [Launcher](https://github.com/starcounterapps/Launcher) to run simultaneously to work properly. It will only display the user interface for users that are signed in. See the [instructions for the SignIn app](https://github.com/starcounterapps/signin#default-admin-user) to create a first user. 
+* Starcounter version 2.3.0.6067 or later
+* A signed in user
```
Comment (warpech) [inline README.md]: I would change the formatting to this:

* A signed in user
   * The first user can be created and signed in using the [SignIn](https://github.com/starcounterapps/signin) app. Follow the instructions in the app's [README](https://github.com/starcounterapps/signin#create-the-first-user) to do that.

File: README.md [hunk 3]
```diff
@@ -4,9 +4,15 @@ Address book. Manage contact information and define relations between **people**
...
+* An app that wraps the HTML templates
```
Comment (warpech) [inline README.md]: Is this true? Each app should be able to run standalone.

You can have two apps in standalone (People and SignIn) but then you will just need to get to then using the URLs:

- http://localhost:8080/signin/signinuser
- http://localhost:8080/people

Pls check if that works for you

File: README.md (hunk 2 above)
Comment (Mackiovello) [inline README.md]: Agree

File: README.md (hunk 3 above)
Comment (Mackiovello) [inline README.md]: > Is this true? Each app should be able to run standalone.

Well, the thing is that if you build and run SignIn and People without being signed in and go to `http://localhost:8080/people`, nothing will be stamped. The source will look like this:

```html
<link rel="import" href="/sys/polymer/polymer.html">

<template>
    <template is="dom-bind">
        <div slot="people/Authorization-failed">You need to be signed in to use People.</div>
    </template>
</template>
```

So the issue is that the authorization page is not wrapped properly. You should be able to see what's wrong, even if you're not signed in.

The solution of going to `signin/signinuser` and then `/people` works.  I'll include that in the instructions.

File: README.md (hunk 1 above)
Comment (Mackiovello) [inline README.md]: Okay, I'll revert it

File: README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ Address book. Manage contact information and define relations between **people**
 
 Requires Starcounter version 2.3.0.6067
 
-This app requires [SignIn](https://github.com/starcounterapps/signin) 4.0.0 or newer and the [Launcher](https://github.com/starcounterapps/Launcher) to run simultaneously to work properly. It will only display the user interface for users that are signed in. See the [instructions for the SignIn app](https://github.com/starcounterapps/signin#default-admin-user) to create a first user. 
+The app also requires a user to be signed in. This can be done with the [SignIn](https://github.com/starcounterapps/signin) app. If there are no users in the database, [create the first user](https://github.com/starcounterapps/signin#create-the-first-user). Otherwise, [sign in an existing user](https://github.com/starcounterapps/signin#sign-in)
 
 For developer instructions, go to [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (warpech) [inline README.md]: Can we move the developer instructions to the bottom of README.md, just like here: https://github.com/StarcounterApps/Website/blob/develop/README.md

**Comment by Mackiovello** [comment]
Moved the instructions down and added a headline. Merging

**Pull request merged**